var/spack/repos/builtin/packages/elpa/package.py

```diff
@@ -113,7 +124,8 @@ def configure_args(self):
             'FC={0}'.format(spec['mpi'].mpifc),
             'CXX={0}'.format(spec['mpi'].mpicxx),
             'LDFLAGS={0}'.format(spec['lapack'].libs.search_flags),
-            'LIBS={0}'.format(spec['lapack'].libs.link_flags),
+            'LIBS={0} {1}'.format(
+                spec['lapack'].libs.link_flags, spec['blas'].libs.link_flags),
```

Comment:
does this work when you use e.g. openblas which bundles lapack and blas in a single library?

Reply:
@haampie , yes it works.